src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -133,21 +138,19 @@ public String getName()
         return "Integrity Check Spark Job";
     }
 
-    @SuppressWarnings({ "rawtypes", "unchecked" })
+    @SuppressWarnings("unchecked")
     @Override
     public void start(final CommandMap commandMap)
     {
-        final String atlasDirectory = (String) commandMap.get(ATLAS_FOLDER);
+        final String atlasDirectory = (String) commandMap.get(SparkJob.INPUT);
```

Comment:
`public static final Switch<String> INPUT = new Switch<>("input", "Input path of the Spark Job", StringConverter.IDENTITY, Optionality.OPTIONAL);` in `org.openstreetmap.atlas.generator.tools.spark.SparkJob`. Notably, `ATLAS_FOLDER` has been deprecated since atlas-checks 6.0.2, and I didn't see any differences in my runs when I switched it to the recommended switch.

Reply:
Ah, thanks!